src/main/webapp/app/sentry/sentry.error-handler.ts

```diff
@@ -19,16 +19,18 @@ export class SentryErrorHandler implements ErrorHandler {
     }
 
     private async initSentry(): Promise<void> {
-        const profileInfo = await this.profileService.getProfileInfo();
+        this.profileService.getProfileInfo().subscribe((profileInfo: ProfileInfo) => {
+            if (profileInfo) {
+                if (!profileInfo.sentry) {
```

Comment:
Can we simplify this change with something like this:
```suggestion
                if (! profileInfo || !profileInfo.sentry) {
```

Reply:
good point. i'll change it